apps/dashr-ncbi-explorer/app.R

```diff
@@ -0,0 +1,654 @@
+appName <- Sys.getenv("DASH_APP_NAME")
+if (appName != "") {
+  pathPrefix <- sprintf("/%s/", appName)
+  
+  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+}
+
+# set working directory
+setwd("/app")
+
+library(dash)
+library(dashBio)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(Biostrings)
+library(readr)
+library(ape)
+library(stringr)
+library(rentrez)
+library(dashTable)
+library(plotly)
+
+app <- Dash$new()
+
+# Functions and Data Loading
+
+FASTA_DATA <- readBStringSet(filepath = "data/alignment_viewer_p53_clustalo.FASTA")
+
+blank_dataframe <- data.frame(matrix(ncol=0,nrow=0))
+
+generate_table <- function(df, nrows=20)
```

Comment:
we could place all the functions in a `helpers.R` for transparency and `source` them